README.md

```diff
@@ -982,7 +982,7 @@
     // declaration to the top of the scope,
     // which means our example could be rewritten as:
     function example() {
-      let declaredButNotAssigned;
+      var declaredButNotAssigned;
```

Comment:
Alright, it's just a little bit strange for me, however, I will change it back to `let`.